src/main/resources/application.properties

```diff
@@ -60,3 +60,7 @@ gingersnap.region.us-east.backend.key-columns=fullname
 
 #quarkus.log.category."io.debezium.connector.sqlserver".level=TRACE
 #quarkus.log.category."io.debezium.connector.sqlserver".min-level=TRACE
+
+## Kubernetes Configuration
+gingersnap.k8s.rule-name=
```

Comment:
I think we need to rename this property. The value is the name of the ConfigMap to be watch which contains the individual rule definitions. I think this property should be something like rule-config-map.

There should then probably be an optional property, something like `gingersnap.k8s.rules=rule1,rule2` that determines which specific rules are processed.

Reply:
Updated the name.